src/Internal/Wallet/Cip30.js

```diff
@@ -5,16 +5,29 @@ exports._getNetworkId = conn => () => conn.getNetworkId();
 exports._getUtxos = maybe => conn => () =>
   conn.getUtxos().then(res => (res === null ? maybe.nothing : maybe.just(res)));
 
+const getCollateral = conn =>
+  conn.getCollateral !== undefined
```

Comment:
@klntsky 

I did not think that we can do that.

If so we can drop experimental prefix for MockCip30?

https://github.com/Plutonomicon/cardano-transaction-lib/issues/1133

Reply:
@uhbif19 Nami only implements `getCollateral` in the experimental namespace, if we want to support Nami, we have no choice but to use it like this for `getCollateral`. Maybe it would be worth to have a comment explaining this.

Reply:
Whether we drop `experimental` from Cip30Mock does not matter as long as we support both methods. Cip30Mock is just for testing, after all. I advise not to change it, since it would not provide any value.